examples/bri-3/src/bri/workgroup/workgroups/agents/workgroupStorage.agent.ts

```diff
@@ -108,6 +107,25 @@ export class WorkgroupStorageAgent extends PrismaService {
     return this.mapper.map(updatedWorkgroupModel, Workgroup, Workgroup);
   }
 
+  async archiveWorkgroup(workgroup: Workgroup): Promise<Workgroup> {
```

Comment:
i dont think we need a separate method for storage, could we just call the update workgroup method? In case there is a large number of entires in the workgroup table we could think about optimizing the UPDATE statement, but currently we are fine to use a single general one..